### PR TITLE
Add whitespace to tags

### LINKED ISSUE #119: Add whitespace to tags

### PR DESCRIPTION
**Pull request by SeanFoongjt**
Fixes #119

### CODE REVIEW

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "emergency name": "Aiken",
     "emergency phone": "98765431",
     "graduation year": "2030",
-    "tags" : [ "owesMoney", "friends" ]
+    "tags" : [ "owes Money", "friends" ]
```
Comment (chashaobao123) [inline src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json:22]: should we still be using camel case? would it be better if it was "owes money"

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 2]
```diff
@@ -33,7 +34,7 @@ public class TypicalPersons {
             .withAddress("311, Clementi Ave 2, #02-25").withRoomNumber("05-0523")
             .withEmail("johnd@example.com").withPhone("98765432")
             .withEmergencyContact("Aiken", "98765431").withGradYear("2030")
-            .withTags("owesMoney", "friends").build();
+            .withTags("owes Money", "friends").build();
```
Comment (chashaobao123) [inline src/test/java/seedu/address/testutil/TypicalPersons.java:37]: is there mismatch with the VALID_TAG_OWES_MONEY: supposed to be "owes money"? or idk isit case sensitive?

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json (hunk 1 above)
Comment (SeanFoongjt) [reply src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json]: Technically it does not matter, but it does look funny so I changed it.

**Review by EdaYeo — APPROVED** [review]
LGTM!

**Pull request merged**